### ISSUE TITLE
Add a11y custom action to reader mode button

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#21347]
see `TabLocationView` for old implementation

The a11y custom action should be configurable and optional for toolbar actions.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-9709)

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Verified as fixed on v9000 (45556) with iPhone 15 Pro (17.5).

The reader view button can be selected without issues with voice-over and it is read correctly.

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Verified as fixed on v133 (47038) with iPhone 15 Pro (18.2).
